src/context/UserInfoSummary/index.tsx

```diff
@@ -0,0 +1,48 @@
+import FullPageError from 'components/fullPage/Error';
+import useUserInfoSummary from 'hooks/useUserInfoSummary';
+import { useEffect } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { BaseComponentProps } from 'types';
+import { useUserInfoSummaryStore } from './useUserInfoSummaryStore';
+
+const UserInfoSummaryStoreProvider = ({ children }: BaseComponentProps) => {
+    const { data, isLoading, isValidating, error, mutate } =
+        useUserInfoSummary();
+
+    const populateAll = useUserInfoSummaryStore((state) => state.populateAll);
+    const setMutate = useUserInfoSummaryStore((state) => state.setMutate);
+
+    useEffect(() => {
+        if (data) {
+            populateAll(data);
+        }
+    }, [data, populateAll]);
+
+    useEffect(() => {
+        setMutate(mutate);
+    }, [mutate, setMutate]);
```

Comment:
Kept these separate for now mainly because I am getting a bit suspicious of storing things like mutate inside the store.